pocs/camera/camera.py

```diff
@@ -496,14 +480,44 @@ def _process_fits(self, file_path, info):
         fits_utils.update_headers(file_path, info)
         return file_path
 
+    def _create_subcomponent(self, subcomponent, name, class_name, base_class):
+        """
+
```

Comment:
Maybe a quick docstring about what this does.

Reply:
OK.